redfish-core/lib/bios.hpp

```diff
@@ -476,12 +525,92 @@ inline void requestRoutesBiosSettings(App& app)
                                                          attrName);
                         return;
                     }
-                    int64_t attrValue = attrItr.value();
+                    const int64_t& attrValue = attrItr.value();
+
+                    boost::container::flat_map<
+                        std::string, std::variant<int64_t, std::string>>
+                        attrBaseOptionsMap = std::get<2>((*it).second);
+
+                    int64_t lowerBoundVal = 0;
+                    int64_t upperBoundVal = 0;
+
+                    // Get Lower Bound value
+                    auto iter = attrBaseOptionsMap.find("LowerBound");
+                    if (iter != attrBaseOptionsMap.end())
+                    {
+                        lowerBoundVal = std::get<int64_t>((*iter).second);
+                    }
+
+                    // Get Upper Bound value
+                    iter = attrBaseOptionsMap.find("UpperBound");
+                    if (iter != attrBaseOptionsMap.end())
+                    {
+                        upperBoundVal = std::get<int64_t>((*iter).second);
+                    }
+
+                    if (attrValue < lowerBoundVal || attrValue > upperBoundVal)
+                    {
+                        BMCWEB_LOG_ERROR << "Attribute value is out of range";
```

Comment:
{[m
                         BMCWEB_LOG_ERROR << "Attribute value is out of range";[m
[31m-                        std::string val =[m
[31m-                            boost::lexical_cast<std::string>(attrValue);[m
[32m+[m[32m                        std::string val = std::to_string(attrValue);[m
                         messages::propertyValueOutOfRange(asyncResp->res, val,[m
                                                           attrName);[m
                                                           
Take out that boost::lexical_cast ?